image/nitf.ksy

```diff
@@ -0,0 +1,766 @@
+meta:
+  id: nitf
+  title: National Imagery Transmission Format
+  file-extension:
+    - ntf
+    - nitf
+    - ntf.r0
+    - ntf.r1
+    - ntf.r2
+    - ntf.r3
+    - ntf.r4
+    - ntf.r5
+  ks-version: 0.9
+  encoding: UTF-8
+  endian: be
+  license: MIT
+doc: |
+  Is set to version format (file_version) of 02.10 and standard_type of BF01. Implemented by River Loop Security.

```

Comment:
@rmspeers Can you please add a few sentences describing the format here? What it is used for? What kind of data it stores? What is the main usage field?

The point is for someone who visits the format page in our format gallery (e.g. [ASN.1 DER](https://formats.kaitai.io/asn1_der/index.html) format) to get a basic idea about what kind of the format it is, and not have to search it on Wikipedia or other external websites.

Reply:
Added, see if that is what you were looking for?

Reply:
> Added, see if that is what you were looking for?

Yes, that's better 👍